docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        tgts = np.where(lbMP &lt;= bsf)[0]
</pre><p>Do you mean <code>targets</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>